examples/realtime/ftclient_rt_compute_psd.py

```diff
@@ -43,7 +43,7 @@
     n_samples = 2048  # time window on which to compute FFT
     for ii in range(20):
         epoch = rt_client.get_data_as_epoch(n_samples=n_samples, picks=picks)
-        psd, freqs = compute_epochs_psd(epoch, fmin=2, fmax=200, n_fft=n_fft)
+        psd, freqs = psd_welch(epoch, fmin=2, fmax=200, n_fft=n_fft)
```

Comment:
sorry my bad, the old link doesn't work any more. I'll send you a copy over email so you can test.